tests/lmeval/test_lmeval.py

```diff
@@ -7,10 +7,20 @@
 import yaml
 from loguru import logger
 
+from pydantic import BaseModel
 from llmcompressor.core import active_session
 from tests.e2e.e2e_utils import run_oneshot_for_e2e_testing
 from tests.examples.utils import requires_gpu_count
 
+
+class LmEvalConfig(BaseModel):
```

Comment:
I could refactor the entire model config to this Pydantic style, with defaults and docstrings it might help for readers / people extending in future

Reply:
I don't see a huge need, as the config should populate most of these and we should throw an error if the non-defaulting ones are skipped.

I do see a need to use 4-space indent though